js/components/Marathon.js

```diff
@@ -326,7 +333,9 @@ var Marathon = React.createClass({
         {
           error: function (data, response) {
             var msg = response.responseJSON.message || response.statusText;
+            /*eslint-disable no-alert, no-console */
             alert("Not scaling: " + msg);
+            /*eslint-enable no-alert */
```

Comment:
Same thing with all of these `alerts` we can have a util that'll do it for us: `User.alert`